src/main/app/form/validation.ts

```diff
@@ -187,13 +189,16 @@ export const isEmailValid: Validator = value => {
 };
 
 export const isFieldLetters: Validator = value => {
-  if (!(value as string).match(/^[\p{Script=Latin}'’\-\s]*$/gu)) {
+  const matcher = new RegExp(/^[\p{Script=Latin}'’\-\s]*$/gu);
```

Comment:
this issue has not been fixed as allowing latin charecter is also not working in aat.Solution to fix this issue is use following script: 
const matcher = new RegExp(/^[\p{L}'’\-\s]*$/gu); instead of const matcher = new RegExp(/^[\p{Script=Latin}'’\-\s]*$/gu);